backend/Api/Options/OrganizationOptions.cs

```diff
@@ -0,0 +1,47 @@
+
+namespace Api.Options;
+
+public class OrganizationOptions
+{
+    private static OrganizationOptions _organizationOptions;
+
+    public double HoursPerWorkday { get; set; }
+
+    public OrganizationOptions(IConfiguration config)
+    {
+        this.HoursPerWorkday = config.GetValue<double>("HoursPerWorkday");
+
+        // Now set Current
+        _organizationOptions = this;
+    }
+
+    public static OrganizationOptions Current
+    {
+        get
+        {
+            if (_organizationOptions == null)
+            {
+                _organizationOptions = GetCurrentSettings();
+            }
+
+            return _organizationOptions;
+        }
+    }
+
+    public static OrganizationOptions GetCurrentSettings()
+    {
+        var env = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
+        
+        var builder = new ConfigurationBuilder()
+            .SetBasePath(Directory.GetCurrentDirectory())
+            // .AddJsonFile($"appsettings.{env}.json", optional: true, reloadOnChange: true)
+            .AddJsonFile("appsettings.Local.json", optional: true, reloadOnChange: true)
+            .AddEnvironmentVariables();
+
+        var configuration = builder.Build();
+
+        var settings = new OrganizationOptions(configuration.GetSection("OrganizationSettings"));
```

Comment:
The problem with this implementation is that is we want to add sources for configuration, say keyvault or secrets.json file, then we need to manually add that here (and in any other Options-class for that matter)

This class should instead be implemented as a singleton, where built config in taken into the ctor

Reply:
That I can fix in a followup-task: Will create an issue for this now 👏